Assets/Scripts/Utility/RMBLayout.cs

```diff
@@ -348,44 +348,45 @@ public static GameObject CreateBaseGameObject(ref DFBlock blockData, int layoutX
                     obj.XPos,
                     -obj.YPos + blockFlatsOffsetY,
                     obj.ZPos + BlocksFile.RMBDimension) * MeshReader.GlobalScale;
+                
+                GameObject go = MeshReplacement.ImportCustomFlatGameobject(obj.TextureArchive, obj.TextureRecord, billboardPosition, flatsParent);
+                if (go == null)
+                {
+                    //// Use misc billboard atlas where available
+                    //if (miscBillboardsAtlas != null && miscBillboardsBatch != null)
+                    //{
+                    //    TextureAtlasBuilder.AtlasItem item = miscBillboardsAtlas.GetAtlasItem(obj.TextureArchive, obj.TextureRecord);
+                    //    if (item.key != -1)
+                    //    {
+                    //        miscBillboardsBatch.AddItem(item.rect, item.textureItem.size, item.textureItem.scale, billboardPosition);
+                    //        continue;
+                    //    }
+                    //}
+
+                    // Add to batch where available
+                    //if (obj.TextureArchive == TextureReader.AnimalsTextureArchive && animalsBillboardBatch != null)
+                    //{
+                    //    animalsBillboardBatch.AddItem(obj.TextureRecord, billboardPosition);
+                    //    continue;
+                    //}
```

Comment:
Do we need to keep this commented out code around anymore? Seems to have been commented out 4 years ago when some super atlas was disabled.

Reply:
Yeah, the commented code can be edited out. I'll remove it after I merge.